tests/test_tabular.py

```diff
@@ -74,6 +75,7 @@ def test_random_mdp():
 def test_policy_om_random_mdp():
   """Test that optimal policy occupancy measure ("om") for a random MDP makes
   sense."""
+  mdp = gym.make('imitation/Random-v0')
```

Comment:
Comment (e.g. `# test env is registered under some name`) to make overwrite of `mdp` less confusing.

Reply:
I still don't understand why you need initialize RandomMDP through both `gym.make` and `RandomMDP.__init__`. Doesn't `gym.make` just call `RandomMDP.__init__` in the first place?

Reply:
You're right, I don't need the `RandomMDP` call any longer.